Packages/vcs/Lib/VTKPlots.py

```diff
@@ -357,7 +358,7 @@ def createRenderer(self, *args, **kargs):
         return ren
 
     def update(self, *args, **kargs):
-        self._lastSize = -1
+        self._lastSize = None
```

Comment:
@chaosphere2112 For test_vcs_missing_colorname I get 'Cell index must be in the range 0 to 255'. I'll let you fix that.